README.md

```
@@ -0,0 +1,140 @@
+# Findex
+
+금융위원회 Open API 기반 한국 주가지수 분석 백엔드 서비스
+
+> 공공데이터 Open API 연동, 이동평균·등락률 금융 지표 계산, Spring Scheduler 자동 수집 파이프라인, Railway.io 실서비스 배포까지 포함한 백엔드 프로젝트
+
+<details>
+  <summary>Table of Contents</summary>
+
+- [Contributors](#contributors)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)